comment_scan/chatter.rb

```diff
@@ -21,7 +21,8 @@ def initialize(chatXuser, chatXpwd, hqroom, logger, rooms)
         @mention_actions = [] 
         @fall_through_actions = [] 
 
-        #TODO: mention_received logic will be run alongside both command and reply logic. Going to need to fix this at some point
+        #Note: mention_received logic will be run alongside command logic.
+        #       Since they're run in threads, it'll be hard to prevent that 
```

Comment:
You're welcome to make it single threaded. It really doesn't get enough volume to need to be multithreaded. (just a random thought, you're also welcome to leave it)